packages/vscode-extension/src/debugging/DebugAdapter.ts

```diff
@@ -96,6 +98,7 @@ export class DebugAdapter extends DebugSession {
     this.absoluteProjectPath = configuration.absoluteProjectPath;
     this.projectPathAlias = configuration.projectPathAlias;
     this.connection = new WebSocket(configuration.websocketAddress);
+    this.lineOffset = configuration.lineOffset;
```

Comment:
This parameter should really be called `expoPreludeExtraLines` or something along these lines instead of being so generic